helm/templates/statefulset.yaml

```diff
@@ -1,43 +1,43 @@
+{{- $name := include "immudb.fullname" . }}
+{{- $names := list $name }}
 {{- if gt (.Values.replicaCount | toString | atoi) 1 }}
-{{- fail "At the moment, you can just have 1 instance of immudb. We are working to raise that limit."}}
+  {{- $names = append $names (printf "%s-rr" $name) }}
 {{- end }}
+
+{{- range $names }}
+{{- $isMaster := eq . $name }}
+---
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: {{ include "immudb.fullname" . }}
+  name: {{ . }}
   labels:
-    {{- include "immudb.labels" . | nindent 4 }}
+    {{- include "immudb.labels" $ | nindent 4 }}
 spec:
-  replicas: {{ .Values.replicaCount }}
+  replicas: {{ $isMaster | ternary 1 (sub $.Values.replicaCount 1) }}
   selector:
     matchLabels:
-      {{- include "immudb.selectorLabels" . | nindent 6 }}
-  serviceName: {{ include "immudb.fullname" . }}    
+      {{- include ($isMaster | ternary "immudb.selectorLabels" "immudb.rrSelectorLabels") $ | nindent 6 }}
+  serviceName: {{ $name }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
+      {{- with $.Values.podAnnotations }}
       annotations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
-        {{- include "immudb.selectorLabels" . | nindent 8 }}
+        {{- include ($isMaster | ternary "immudb.selectorLabels" "immudb.rrSelectorLabels") $ | nindent 8 }}
     spec:
-      {{- with .Values.imagePullSecrets }}
+      {{- with $.Values.imagePullSecrets }}
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      securityContext:
-        {{- toYaml .Values.podSecurityContext | nindent 8 }}
-      volumes:
-      - name: immudb-storage
```

Comment:
I see the volume removed but where it has been moved to? I can't find the place.

Reply:
If I'm not mistaken, that's not needed (see example [here](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components))

Reply:
It is not needed *if* you provide a volumeClaimTemplate. You must provide one when using more than one replicas